frontend/javascripts/oxalis/controller.js

```diff
@@ -140,7 +143,9 @@ class Controller extends React.PureComponent<PropsWithRouter, State> {
 
     app.vent.trigger("webknossos:ready");
     Store.dispatch(wkReadyAction());
-    this.setState({ ready: true });
+    setTimeout(() => {
```

Comment:
Sorry, I should have added a comment. I did that now, the comment is:

```
// Give wk (sagas and bucket loading) a bit time to catch air before
// showing the UI as "ready". The goal here is to avoid that the
// UI is still freezing after the loading indicator is gone.
```

I reduced the timeout to 200 ms. It's quite arbitrary and we could also remove the timeout altogether, but from my experience wk is still a bit unresponsive after "set ready" is dispatched, so my hope is that the timeout makes the experience a bit smoother.

Reply:
Thanks, that makes sense :)